scripts/setup-dev.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=invalid-name
+import sys
+import os
+import signal
+import subprocess
+
+children = []
+
+try:
+    owd = os.getcwd()
+
+    d = ""
+    noAuthPath = ""
+
+    print("Running script to setup local Arlo instance...")
+
+    res = input("Do you have an instance of nOAuth installed? [y/N]")
```

Comment:
any reason not just to use the VotingWorks nOAuth instance that's hosted on Heroku? the only reason I can think of is running Arlo offline, but it seems unlikely that someone would need to do that

Reply:
Maybe offer an option for doing either? If someone wants to start a completely new setup from scratch it could make sense for them to want to have all the pieces under their control. Also from a dev perspective, it's nice to be able to have the option to code even when on the road or wherever.

Reply:
I'll mess around with it. I just wrote this after the way I use nOAuth. I didn't know we had web instance :sweat_smile: